modules/database/sqlite/load.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Module Name: SQLite Integration
+ * Description: Use an SQLite database instead of MySQL.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Require the site-health tweaks.
+ */
+require_once __DIR__ . '/site-health.php';
+
+/**
+ * Add admin notices.
+ *
+ * When the plugin gets merged in wp-core, this is not to be ported.
+ *
+ * @since n.e.x.t
+ */
+function perflab_sqlite_plugin_admin_notice() {
+	// Check if the wp-content/db.php file exists.
+	if ( ! file_exists( WP_CONTENT_DIR . '/db.php' ) ) {
```

Comment:
See my other comment; this is not really reliable, can we instead check for a drop-in specific constant here to know it's in fact _our_ `db.php` that has been loaded?

Reply:
Fixed 👍